modules/apps/portal-crypto-hash-provider/portal-crypto-hash-provider-message-digest/configs/com.liferay.portal.crypto.hash.provider.message.digest.internal.configuration.MessageDigestCryptoHashProviderConfiguration-default.config

```diff
@@ -0,0 +1,3 @@
+configuration.name="Default Message Digest Crypto Hash Provider Configuration"
```

Comment:
It looks this pull is introducing a bundled configuration file
here.


If that is the case and this pull is fixing a bug please mark this pull as 
*DO NOT BACKPORT*


Please see https://issues.liferay.com/browse/PTR-1770